client/src/components/main/chatwindow/EditComment.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import Styled from './styles/comment.styles';
+import axios from 'axios';
+import { ChatContext } from './ChatWindow';
+
+/*
+Posts new comment to database when enter key pressed and triggers re-render by updating ChatWindow state
+*/
+
+const EditComment = props => {
+  const { _id, className, setEditComment, children } = props;
+  const [comment, setComment] = useState(children);
+  const { dispatch } = useContext(ChatContext);
+
+  const handleSubmit = text => {
+    axios
+      .patch(
+        `/api/comments/${_id}`,
+        {
+          user: localStorage.userId,
+          text: text,
+        },
+        {
+          headers: { authorization: `bearer ${localStorage.authToken}` },
+        },
+      )
+      .then(() => {
+        dispatch({ type: 'PATCH_TO_DB', text });
+        setEditComment(false);
+      })
+      .catch(err => console.error(err));
+
+    setComment('');
+  };
+
+  const handleEnter = event => {
+    if (event.keyCode === 13) {
+      event.preventDefault();
+      handleSubmit(comment);
+    }
+  };
+
+  const handleOnChange = event => {
+    setComment(event.target.value);
+  };
+
+  return (
+    <Styled.CommentFormWrapper className={className}>
+      <Form onSubmit={handleSubmit}>
+        <TextArea onChange={handleOnChange} onKeyDown={handleEnter} value={comment}></TextArea>
```

Comment:
You don't need `</TextArea>` in the end, you can immediately close it with `/>` at the end